components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,172 @@
+# Lane Follower for a single camera
+# Applies a perspective transform to an image, and isolates lane lines
+import numpy as np
+import math
+import cv2
```

Comment:
Imports should be sorted alphabetically and have whitespace between a standard lib imports and 3rd-party improts. They should also come before 3rd-party imports.

Reply:
Used isort